src/main/java/com/backbase/ct/bbfuel/client/user/UserPresentationRestClient.java

```diff
@@ -46,4 +58,38 @@ public UserGetResponseBody getUserByExternalId(String userExternalId) {
             .as(UserGetResponseBody.class);
     }
 
+    public void createIdentityUserAndLogResponse(UsersPostRequestBody user){
+
+        Response response = createIdentity(user);
+
+
+        if (isBadRequestException(response, "User already exists")) {
+            log.info("User [{}] already exists, skipped ingesting this user", user.getExternalId());
+        } else if (response.statusCode() == SC_CREATED) {
+            log.info("Identity User [{}] ingested under legal entity [{}]",
+                user.getExternalId(), user.getLegalEntityExternalId());
+        } else {
+            response.then()
```

Comment:
what's this?

Reply:
The majority of the logging method was implemented by someone else, No idea why it ends in this way but thought  its best to not change it

Reply:
line 72

Reply:
This is just a fancy way of throwing an exception when the status isn't created, right?

Reply:
yes? I think so, if it isnt caught in the first two 'if' statements it'll definitely fail the else block and throw up the error